vsintegration/src/FSharp.Editor/IndentationService.fs

```diff
@@ -46,15 +46,20 @@ type internal FSharpIndentationService() =
                      | _ -> spaces
             Some(loop 0 0)
 
+    // TODO: post beta5, update implementation to use ISynchronousIndentationService instead to guarentee Indentation is UI-blocking
```

Comment:
Do we have a bug tracking this TODO?

Given this is a feature branch we may not need bugs for every TODO.  But we should use a phrase that is grepable like FWORKSPACES.  That way before RI'ing we can do a quick grep on the code base for all FWORKSPACE instances and make sure they were properly handled / converted to issues.